packages/core/src/_test/constants.ts

```diff
@@ -247,7 +248,7 @@ const usdcContractAbi = [
 
 export const usdcContractConfig = {
   chainId: 1,
-  address: "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+  address: checksumAddress("0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48"),
```

Comment:
Checking my understanding - this is necessary because normally, these values have already passed through our validator/formatter?

Reply:
Yes